velox/common/caching/AsyncDataCache.cpp

```diff
@@ -195,19 +190,17 @@ CachePin CacheShard::initEntry(
       }
     }
   }
-  entry->touch();
+  entry->accessStats_.reset();
```

Comment:
This code suggests that AsyncDataCacheEntry may be reused, but I don't understand how would that happen. Would you explain?